src/chi/components/picker/picker.scss

```diff
@@ -146,15 +146,15 @@ legend {
   }
 }
 
-.m-picker {
+.chi-picker__item {
```

Comment:
A lot of duplicate selectors. This is throwing `no-mergeable-selectors` errors in the console. Should be able to merge all this into line 25.

Reply:
Sure